velox/dwio/parquet/tests/ParquetWriterTestBase.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/parquet/tests/ParquetWriterTestBase.h"
+
+namespace facebook::velox::parquet {
+
+std::vector<RowVectorPtr> ParquetWriterTestBase::createBatches(
+    const RowTypePtr& rowType,
+    uint64_t numBatches,
+    uint64_t vectorSize) {
+  std::vector<RowVectorPtr> batches;
+  for (auto i = 0; i < numBatches; ++i) {
+    VectorFuzzer fuzzer({.vectorSize = vectorSize}, leafPool_.get());
+    fuzzer.fuzzInputFlatRow(rowType);
```

Comment:
Why call fuzzInputFlatRow and drop the results?

Reply:
Fixed